nx-X11/programs/Xserver/randr/randr.c

```diff
@@ -309,7 +387,13 @@ RRFreeClient(void *data, XID id)
 
     pRREvent = (RREventPtr) data;
     pWin = pRREvent->window;
+#ifndef NXAGENT_SERVER
+    dixLookupResourceByType((void **) &pHead, pWin->drawable.id,
+                            RREventType, serverClient, DixDestroyAccess);
+#else                           /* !defined(NXAGENT_SERVER) */
     pHead = (RREventPtr *) LookupIDByType(pWin->drawable.id, RREventType);
+#endif                          /* !defined(NXAGENT_SERVER) */
+
```

Comment:
Those lookups are called deprecated here: https://cgit.freedesktop.org/xorg/xserver/commit/randr/randr.c?id=faeebead7bfcc78535757ca7acc1faf7554c03b7

So _maybe_ we can switch those right now to the new way?